Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -227,53 +421,383 @@ public void setSuffix(String suffix) {
         this.suffix = suffix;
     }
 
+    public String getDialogTitle() {
+        return dialogTitle;
+    }
+
+    public void setOrganizationId(String organizationId) {
+        this.organizationId = organizationId;
+    }
+
+    public String getOrganizationId() {
+        return organizationId;
+    }
+
+    public void setIdentifierPatientID(String identifierPatientID) {
+        this.identifierPatientID = identifierPatientID;
+    }
+
+    public String getIdentifierPatientID() {
+        return identifierPatientID;
+    }
+
+    public void setStreet1(String street1) {
+        this.street1 = street1;
+    }
+
+    public String getStreet1() {
+        return street1;
+    }
+
+    public void setStreet2(String street2) {
+        this.street2 = street2;
+    }
+
+    public String getStreet2() {
+        return street2;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setPostal(String postal) {
+        this.postal = postal;
+    }
+
+    public String getPostal() {
+        return postal;
+    }
+
+    public void setPhonenumberValue(String phonenumberValue) {
+        this.phonenumberValue = phonenumberValue;
+    }
+
+    public String getPhonenumberValue() {
+        return phonenumberValue;
+    }
+
+    // IDs and isValidId
+    private boolean isValidPatientId() {
+        return withPatient != null && HelperUtil.isId(withPatient.getPatientId());
+    }
+
     public Long getPatientId() {
-        return updatePatientId;
+        if (isValidPatientId()) {
```

Comment:
Refactor the get Id code to GenericDAO to remove the redundant logic.

Reply:
there is no common interface in the entity-id